POSserver/server/schemas/job.py

```diff
@@ -31,7 +31,7 @@ class Arguments:
         description = graphene.String()
 
     ok = graphene.Boolean()
-    job_field = graphene.Field(Job_Type)
+    job = graphene.Field(Job_Type)
 
```

Comment:
Is this another use of **kwargs because Django docs demanded it?

Reply:
Without **kwargs, the query doesn’t accept arguments from the user. I believe I attempted to remove them per your earlier question regarding them and found that to be the case.

Reply:
Thanks Amy..was just curious...